chaos_monkey/system_info/__init__.py

```diff
@@ -46,7 +46,11 @@ def __init__(self):
         self.info = {}
 
     def get_hostname(self):
-        self.info['hostname'] = socket.gethostname()
+        """
+        Adds the computer hostname to the system information.
+        :return:
```

Comment:
Documentation is good, but if you already did the effort, and wrote on get_network_info that you return None, do it here as well :)